config/routes.rb

```diff
@@ -1,6 +1,12 @@
 Rails.application.routes.draw do
+  # get 'users/index'
   # Define your application routes per the DSL in https://guides.rubyonrails.org/routing.html
 
   # Defines the root path route ("/")
-  # root "articles#index"
+  root "users#index"
+  get "/users/:id/", to: "users#show"
+
+  get "users/:user_id/posts", to: "posts#index"
+  get "users/:user_id/posts/:id/", to: "posts#show"
```

Comment:
### Optional
- [ ] It would be nice if we use the `resources` for our routes instead of manually nesting posts in users. For example;

```ruby
 resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```
This will help us achieve the same results as we have now but more simpler

Reply:
Done